fiddler/datadog_checks/fiddler/check.py

```diff
@@ -265,6 +283,8 @@ def _run_queries(self, project: str, model: str) -> (Dict[str, Any], list):
         start_time = time.strftime('%Y-%m-%d %H:%M:%S', time.gmtime(start_time_epoch / 1000))
         end_time = time.strftime('%Y-%m-%d %H:%M:%S', time.gmtime(end_time_epoch / 1000))
 
+        baseline_name = self._get_baseline_name(project, model)
```

Comment:
I think it makes more sense to call it only when baseline name is required for the metric. We should move it inside the if condition  - https://github.com/DataDog/integrations-extras/pull/2368/files#diff-c10da9b700759108e912edfd1bab6da1ba032c6679973175e6f8ea898da57ce6R321.

Reply:
Actually, this one is better, because we only get baseline name for a model once instead of for each metric.
Moving this under the if condition will mean multiple calls to get baseline name.